main.rb

```diff
@@ -0,0 +1,47 @@
+require 'date'
+require './app'
+
+def display_menu
+  puts "\nWelcome to School Library App!"
+  puts "\nPlease choose an option by entering a number:"
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
```

Comment:
- [ ] As I could not create a person, I could not also test these create rental and list rental options. Kindly ensure they work too.